### PR TITLE
Updates to about and news banner

### PR DESCRIPTION
**Pull request by jukent**
#16 and #18

### CODE REVIEW

File: content/about/_index.md [hunk 1]
```diff
@@ -13,3 +13,5 @@ The Geoscience Community Analysis Toolkit (GeoCAT) team, established in 2019, le
 GeoCAT aims to create scalable data analysis and visualization tools for Earth System Science data to serve the geosciences community in the scientific Python ecosystem. GeoCAT tools are built upon cornerstone technologies in the Pangeo stack such as Xarray, Dask, and Jupyter Notebooks. Novel research and development is conducted for analyzing and visualizing structured as well as unstructured grid data from various research fields such as climate, weather, atmosphere, ocean, etc. In addition, some of the functionalities in the GeoCAT stack are inspired/reimplemented from NCL (NCAR Command Language).
 
 The GeoCAT team is committed to open development, prioritizing community involvement at all levels of the project lifecycle, from requirements and specification to testing and deployment, alongside having the whole software stack open-sourced.
+
+GeoCAT is a team within NSF NCAR's [Visualization & Analysis Systems Technologies (VAST)](https://vast.ucar.edu/). VAST, and in turn GeoCAT, are within the [Computational and Information Systems Lab (CISL)](https://www.cisl.ucar.edu/). as shown in the [UCAR org chart](https://ncar.ucar.edu/who-we-are/org-chart).
```
Comment (kafitzgerald) [inline content/about/_index.md]:
```suggestion
GeoCAT is a team within NSF NCAR's [Visualization & Analysis Systems Technologies (VAST)](https://vast.ucar.edu/) section of the [Computational and Information Systems Lab (CISL)](https://www.cisl.ucar.edu/).
```

**Review by kafitzgerald — APPROVED** [review]
I added a short suggestion that removes an extra period that seems to have snuck in, is a bit more concise, and removes the org chart link since that only goes down to the CISL level anyway (as far as I can tell).  I think the period probably needs to go, but otherwise this is really personal preference so take or leave whatever.

I admittedly haven't looked over a preview yet, but I trust the second change addresses the issue I logged about the headings getting cut off.

**Comment by jukent** [comment]
Thanks @kafitzgerald I tried various screen sizes (max of my laptop width) and at the new height the text is never cut off.

**Comment by jukent** [comment]
Might not be related, but after the push our CNAME disappeared from pages again.  I re-added it. Will keep watch on if the site deploys.

E: quickly redeployed but not sure why it disappeared again

**Comment by erogluorhan** [comment]
The previous disappearing was also after a PR IIRC, so that's strange!

**Comment by jukent** [comment]
@erogluorhan we fixed it in #21

**Comment by erogluorhan** [comment]
Oops. I'd missed it, thank you!

**Pull request merged**